docs/user-guide/rug/commands.md

```diff
@@ -1,17 +1,19 @@
-Controlling communication and information flows is paramount to effective teams.
-Open-Source projects have long been relying on chat venues to operating and
-performing well. Recent modern chat solutions, which can integrate deeply
-in your ecosystem, have proven to be fantastic central venues for your teams to
-drive their project flows.
+Controlling communication and information flows is paramount to
+effective teams.  Open Source projects have long been relying on chat
+venues to operating and performing well.  Recent modern chat
```

Comment:
Sorry, I just filled the paragraph and did not review or modify it. I'll clean it up.